src/crystal/iconv.cr

```diff
@@ -1,9 +1,21 @@
-require "c/iconv"
+{% if flag?(:win32) %}
```

Comment:
Wasn't there a flag `:msvc`? If you're saying this is specific to MSVC

Reply:
Maybe, but to date nobody seems to really attempt building Crystal on Cygwin or MSYS(2), so to my understanding `flag?(:msvc)` is currently reserved for code that actually invokes the MSVC build tools.